core/app/models/spree/promotion/rules/first_purchase_since.rb

```diff
@@ -0,0 +1,36 @@
+module Spree
+  class Promotion
+    module Rules
+      class FirstPurchaseSince < PromotionRule
```

Comment:
hm yea it's a bit fuzzy as it currently stands. i have a small concern with using the word repeat, since marketers might start to think it's a multiple purchase since? maybe i'm over thinking this....